_inc/client/my-plan/my-plan-header/checklist-progress-card/index.js

```diff
@@ -2,47 +2,66 @@
  * External dependencies
  */
 import { translate as __ } from 'i18n-calypso';
-import React from 'react';
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
+import analytics from 'lib/analytics';
 import Button from 'components/button';
 import Card from 'components/card';
 import ProgressBar from './progress-bar';
 import QueryChecklistProgress from 'components/data/query-checklist-progress';
 import { getSiteRawUrl } from 'state/initial-state';
 import { getChecklistCompletion } from 'state/checklist/selectors';
 
-function ChecklistProgressCard( { completed, total, siteSlug } ) {
-	return (
-		<>
-			<QueryChecklistProgress />
-			{ completed && total && (
-				<Card compact className="checklist__header">
-					<div className="checklist__header-main">
-						<div className="checklist__header-progress">
-							<span className="checklist__header-progress-text">
-								{ __( 'Your Jetpack setup progress', {
-									comment: 'Onboarding task list progress',
+class ChecklistProgressCard extends Component {
+	trackCtaClick = () =>
+		void analytics.tracks.recordEvent(
+			'jetpack_myplan_progresschecklistcta_click',
+			this.props.plan
+				? {
+						plan: this.props.plan,
+				  }
+				: undefined
```

Comment:
Don't we always return a plan, even for free users?

Reply:
We should, in theory. However, I don't trust that the plan prop will always be correctly provided.